sentinel-adapter/sentinel-apache-dubbo-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/dubbo/SentinelDubboProviderFilter.java

```diff
@@ -70,13 +71,32 @@ public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcExcept
         String prefix = DubboAdapterGlobalConfig.getDubboProviderResNamePrefixKey();
         String interfaceResourceName = getInterfaceName(invoker, prefix);
         String methodResourceName = getMethodName(invoker, invocation, prefix);
+        ContextUtil.enter(methodResourceName, origin);
         try {
             // Only need to create entrance context at provider side, as context will take effect
             // at entrance of invocation chain only (for inbound traffic).
-            ContextUtil.enter(methodResourceName, origin);
```

Comment:
The comments could be put before `ContextUtil.enter(methodResourceName, origin);`, making it consistent with the code.